backends/rapidpro/status.go

```diff
@@ -102,9 +104,15 @@ UPDATE msgs_msg SET
 	next_attempt = CASE 
 		WHEN 
 			:status = 'E' 
-		THEN 
-			NOW() + (5 * (error_count+1) * interval '1 minutes') 
-		ELSE 
+		THEN CASE
+			WHEN
+				:next_attempt_interval = '24 hours'
+			THEN
+				NOW() + interval '24 hours'
+			ELSE
+				NOW() + (5 * (error_count+1) * interval '1 minutes')
+			END
```

Comment:
Feels a bit odd that `next_attempt_interval` is only used if it's `'24 hours'` and otherwise ignored. We could pass the number of minutes and have it default to `5 * (error_count+1)`

```sql
NOW() + CASE WHEN :next_attempt_minutes > 0 THEN (:next_attempt_minutes || ' minutes')::interval ELSE (5 * (error_count+1) * interval '1 minutes') END;
```